pkg/images/enricher/enricher.go

```diff
@@ -76,6 +76,8 @@ type EnrichmentContext struct {
 	Namespace string
 
 	Source *RequestSource
+
+	ScannerTypeHint string
```

Comment:
Please add a comment that explains what this does.  example:

```suggestion
	// ScannerTypeHint will ensure this scanner type is used when a scan is executed, 
	// does nothing if a scan is not executed.
	ScannerTypeHint string
```